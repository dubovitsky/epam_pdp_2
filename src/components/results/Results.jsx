import React, { useContext } from 'react';
import { Context } from "context/Context.js";

export default function Results () {
    const [context] = useContext(Context);
    const { total } = context;

    return (
        <div>{total} moview found</div>
    )
}
