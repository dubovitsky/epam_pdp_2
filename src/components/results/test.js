import React from 'react';
import { shallow } from 'enzyme';
import Results from './Results';

describe('Results', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Results debug />);
  
    expect(component).toMatchSnapshot();
  });
});