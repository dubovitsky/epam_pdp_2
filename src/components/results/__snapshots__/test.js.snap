// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results should render correctly in "debug" mode 1`] = `
<div>
  7 moview found
</div>
`;
